### ISSUE TITLE
NameError: global name 'Editor' is not defined

### ISSUE THREAD

**Issue by yantis** [neilagabriel/vim-geeknote#25]
First off. Awesome tool! Thanks for the work you put into it.

It would error out while loading a note. 
To fix that I had to add :
"from geeknote.editor import Editor" to enml.py

Am using the https://github.com/jeffkowalski/geeknote fork of geeknote.

________________________________________________________________________
```
Error detected while processing function Vim_GeeknoteActivateNode:
line    4:
Traceback (most recent call last):
  File "<string>", line 2, in <module>
  File "/home/user/dotfiles/_vim/plugged/vim-geeknote/plugin/vim_geeknote.py", line 35, in GeeknoteActivateNode
    explorer.activateNode(vim.current.line)
  File "/home/user/dotfiles/_vim/plugged/vim-geeknote/plugin/explorer.py", line 374, in activateNode
    node.activate()
  File "/home/user/dotfiles/_vim/plugged/vim-geeknote/plugin/explorer.py", line 256, in activate
    GeeknoteOpenNote(self.note)
  File "/home/user/dotfiles/_vim/plugged/vim-geeknote/plugin/view.py", line 120, in GeeknoteOpenNote
    content = ENMLtoText(note.content)
  File "/home/user/dotfiles/_vim/plugged/vim-geeknote/plugin/enml.py", line 34, in ENMLtoText
    return Editor.ENMLtoText(contentENML)
NameError: global name 'Editor' is not defined
```

**Comment by antte** [comment]
same issue here, thanks for the fix

**Comment by neilagabriel** [comment]
Fixed in 854c365528d43b629b54e0fa99b233588432f859
